app/controllers/account_controllers.rb

```diff
@@ -0,0 +1,46 @@
+
+get '/accounts/login' do
+  erb :login
+end
+
+post '/accounts/login' do
+  user = User.find_by(email: params[:email])
+  db_password = user.password_hash
+  if db_password == params[:password]
+    session[:current_user_id] = user.id
+    redirect ('/accounts/' + user.username)
+  else
+    @error = "Your email or password did not match. Please try again."
+    erb :login
+  end
+end
+
+get '/accounts/:username/edit' do
+  @user = session_current_user
```

Comment:
I would rename `session_current_user` to `current_user` and then just use the `current_user` method from the view.